forge-gui/res/editions/Commander Legends Battle for Baldur's Gate.txt

```diff
@@ -616,5 +616,337 @@ ScryfallCode=CLB
+853 M Xenagos, the Reveler @Jason Chan
+854 U Bloodthirsty Blade @Jason Kang
+855 R Chaos Wand @Kari Christensen
+856 U Dimir Keyrune @Daniel Ljunggren
+857 C Dimir Signet @Raoul Vitale
+858 R Dragon's Hoard @Adam Paquette
+859 U Everflowing Chalice @Steve Argyle
+860 U Fellwar Stone @John Avon
+861 U Hedron Archive @Craig J Spearing
+862 U Herald's Horn @Jason Felix
+863 C Izzet Signet @Raoul Vitale
+864 U Lightning Greaves @Jeremy Jarvis
+865 R Maskwood Nexus @Jason A. Engle
+866 U Mindcrank @Chris Rahn
+867 U Orzhov Signet @Martina Pilcerova
+868 R Phyrexian Revoker @Kev Walker
+869 R Psychosis Crawler @Stephan Martiniere
+870 U Skullclamp @Daniel Ljunggren
+871 U Sol Ring @Mike Bierek
+872 R Solemn Simulacrum @Donato Giancola
+873 R Spellskite @Chippy
+874 R Steel Hellkite @Jaime Jones
+875 R Stuffy Doll @David Rapoza
+876 U Talisman of Creativity @Lindsey Look
+877 U Talisman of Dominance @Mike Dringenberg
+878 U Talisman of Hierarchy @Lindsey Look
+879 U Thought Vessel @rk post
+880 C Ash Barrens @John Avon
+881 U Blighted Woodland @Jason Felix
+882 C Bojuka Bog @Howard Lyon
+883 R Castle Embereth @Jaime Jones
+884 R Castle Locthwain @Titus Lunter
+885 R Castle Vantress @John Avon
+886 R Choked Estuary @Vincent Proce
+887 R Cinder Glade @Adam Paquette
+888 R Creeping Tar Pit @Jason Felix
+889 R Darkwater Catacombs @Cliff Childs
+890 R Desolate Lighthouse @Scott Chou
+891 U Dimir Aqueduct @John Avon
+892 R Drownyard Temple @John Avon
+893 R Exotic Orchard @Steven Belledin
+894 R Game Trail @Adam Paquette
+895 U Gruul Turf @John Avon
+896 C Highland Forest @Alayna Danner
+897 U Izzet Boilerworks @John Avon
+898 R Kessig Wolf Run @Eytan Zana
+899 R Kher Keep @Paolo Parente
+900 C Mortuary Mire @James Paick
+901 R Mossfire Valley @Yeong-Hao Han
+902 R Mosswort Bridge @Jeremy Jarvis
+903 R Mutavault @Fred Fields
+904 U Myriad Landscape @Jonas De Ro
+905 R Nephalia Drownyard @Cliff Childs
+906 U Orzhov Basilica @John Avon
+907 C Path of Ancestry @Alayna Danner
+908 U Port of Karfell @Mauricio Calle
+909 C Prismari Campus @Adam Paquette
+910 R Raging Ravine @Todd Lockwood
+911 U Reliquary Tower @Jesper Ejsing
+912 R River of Tears @Chris J. Anderson
+913 U Rogue's Passage @Christine Choi
+914 R Shambling Vent @Jung Park
+915 C Snowfield Sinkhole @Marta Nael
+916 R Spinerock Knoll @Steve Prescott
+917 U Starlit Sanctum @Ben Thompson
+918 R Sunken Hollow @Adam Paquette
+919 U Tainted Field @Don Hazeltine
+920 U Tainted Isle @Alan Pollack
+921 R Temple of Abandon @Adam Paquette
+922 R Temple of Deceit @Jonas De Ro
+923 R Temple of Epiphany @Adam Paquette
+924 R Temple of Silence @Adam Paquette
+925 U Temple of the False God @James Zapata
+926 U Terrain Generator @Alan Pollack
+927 R Vault of the Archangel @John Avon
+928 R Wandering Fumarole @Florian de Gesincourt
+929 R War Room @Milivoj Ćeran
+930 R Windbrisk Heights @Omar Rayyan
 935 R Wand of Wonder @Lindsey Look
 936 R Elder Brain @Nino Is
+
+[dungeons]
+S Undercity
```

Comment:
@Northmoc did you set up Undercity like the other dungeons and this is needed here?

Reply:
Yep, it's set up the same, so this should be good